getting_started/first_3d_game/03.player_movement_code.rst

```diff
@@ -392,7 +393,7 @@ Preview by clicking the checkbox.
 |image6|
 
 In the top view, make sure your *Camera3D* is selected and move the camera about
-``19`` units on the Z axis (drag the blue arrow).
+``19`` units on the Z axis (drag the blue arrow) then ``9`` units on the Y axis (drag the green arrow).
```

Comment:
This is also not correct, see right below, you *rotate* the camera so it lifts up like this

Reply:
Yes, then you are not following the instructions correctly, so please make sure you've followed *every* step exactly